test/e2e/performanceprofile/cluster-setup/manual-cluster/performance/performance_profile.yaml

```diff
@@ -20,4 +20,4 @@ spec:
   numa:
     topologyPolicy: "single-numa-node"
   nodeSelector:
-    node-role.kubernetes.io/worker-cnf: ""
+    node-role.kubernetes.io/worker-cnf: ""
```

Comment:
Looks like new line got removed at the end of the file here. Let's add that back.

Reply:
done